<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="CaptureTab">
     <attribute name="title">
      <string>Capture</string>
     </attribute>
     <widget class="QPushButton" name="OpenCameraBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Camera</string>
      </property>
     </widget>
     <widget class="ImageDisplayLabel" name="ImageDisplay">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>591</width>
        <height>431</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>Video</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="videoElapsed">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>450</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>00:00:00</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SnapshotBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>snapshot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OpenFileBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PlayBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Play/Pause</string>
      </property>
     </widget>
     <widget class="QListWidget" name="SnapshotList0">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>91</width>
        <height>421</height>
       </rect>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SnapshutTab">
     <attribute name="title">
      <string>Snapshots</string>
     </attribute>
     <widget class="ImageViewListWidget" name="SnapshotList1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget::item
{
    background: rgb(255,255,255); 
}
QListWidget::item:selected
{
    background: rgb(128,128,255);
}</string>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="ImageDisplayLabel" name="ImageEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>591</width>
        <height>431</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>Video</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="idLabel">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>80</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="typeLabel">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>130</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="typeComboBox">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>150</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="saveBtn">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>230</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="ImageDisplayLabel" name="OcrImageDisplay">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>10</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="OcrResultLabel">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>180</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>OCR Result</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="OcrResultEdit">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>200</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="idEdit">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 3</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_Camera"/>
    <addaction name="actionOpen_File"/>
   </widget>
   <widget class="QMenu" name="menu_Setting">
    <property name="title">
     <string>&amp;Setting</string>
    </property>
    <addaction name="actionCapture_File_Resolution"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Setting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Camera">
   <property name="text">
    <string>Open Camera</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionCapture_File_Resolution">
   <property name="text">
    <string>Capture File Resolution</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageDisplayLabel</class>
   <extends>QLabel</extends>
   <header>imagedisplaylabel.h</header>
  </customwidget>
  <customwidget>
   <class>ImageViewListWidget</class>
   <extends>QListWidget</extends>
   <header>imageviewlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
